cleanlab/outlier.py

```diff
@@ -446,12 +446,54 @@ def _get_ood_features_scores(
     # neighbor of each point is the point itself, at a distance of zero.
     distances, _ = knn.kneighbors(features)
 
+    ood_features_scores = transform_distances_to_scores(distances, cast(int, k), t)
+    return (ood_features_scores, knn)
+
+
+def transform_distances_to_scores(distances: np.ndarray, k: int, t: int) -> np.ndarray:
```

Comment:
```suggestion
def _transform_distances_to_scores(distances: np.ndarray, k: int, t: int) -> np.ndarray:
```

Reply:
safer as internal method?  Not sure this should be exposed to users since we want to avoid offering a library with tons of functions

Reply:
> safer as internal method? Not sure this should be exposed to users since we want to avoid offering a library with tons of functions

I extracted this function for it to be shared across the library, outside of the `cleanlab.outlier` module.

The leading underscore indicates that the function only belongs within the `cleanlab.outlier` module, which I don't want.

## Solution

I'll make a `cleanlab.internal.outlier` module and move the function there (without the underscore) so it won't be exposed as a part of the public API. I think that's safe enough.